### PR TITLE
Add fgenesh tool

### PR DESCRIPTION
**Pull request by AnnaSyme**
Need to check button link works ok, and are multiple input file types ok on a single line

### CODE REVIEW

File: webapp/home/labs/genome/annotation.yml [hunk 1]
```diff
@@ -88,6 +88,23 @@ tabs:
               - fasta
             label: Genome assembly
         button_link: "{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fcrs4%2Fprokka%2Fprokka"
+        
+      - title_html: <code>FGenesH</code> - Genome annotation
+        description_html: >
+          <p>
+            <code>FGenesH</code> annotates an assembled genome and outputs a GFF3 file. There are several modules that do different things; search for FGENESH in the tool panel to see them. Note: please apply for access to this tool before use at [this link](https://site.usegalaxy.org.au/request/access/fgenesh).
```
Comment (neoformit) [inline webapp/home/labs/genome/annotation.yml:95]: Using markdown again, should be `<a href="~url~">this link</a>`

File: webapp/home/labs/genome/annotation.yml [hunk 2]
```diff
@@ -88,6 +88,23 @@ tabs:
...
+          </p>
+        inputs:
+          - datatypes:
+              - fasta
+            label: Genome assembly
+          - datatypes:
+              - fasta
+            label: Repeat-masked (hard) genome assembly
+          - datatypes:
+              - .cdna, .pro, .dat
```
[the remark this answers is not in the document]
Comment (neoformit) [reply webapp/home/labs/genome/annotation.yml:105]: I pushed a commit which fixes this - let me know if it's not what you want.

**Review by neoformit — CHANGES_REQUESTED** [review]

**Comment by neoformit** [comment]
@AnnaSyme I check the link and made some changes, here's how it looks. Feel free to edit further if necessary:

image

File: webapp/home/labs/genome/annotation.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (AnnaSyme) [reply webapp/home/labs/genome/annotation.yml]: perfect, thank you! @neoformit

[the remark this answers is not in the document]
Comment (neoformit) [reply webapp/home/labs/genome/annotation.yml]: @AnnaSyme I just checked, and `pro`, `cdna`, `dat` do not seem to be Galaxy datatypes? I can see this confusing people if they try to create these data in Galaxy? Maybe this should be:

```yml
- datatypes:
    - fasta
  label: cDNA or protein sequences for [ ... ]
```

Or maybe this should be omitted altogether, if it's specific to the TSI WF? In the *FGENESH
Genome Annotation* tool I can only see these:

```yml
- datatypes:
    - fasta
  label: map known mRNA data to genomic sequence
- datatypes:
    - fasta
  label: map known ESTs to genomic sequence
```

[the remark this answers is not in the document]
Comment (AnnaSyme) [reply webapp/home/labs/genome/annotation.yml]: hmm yes. perhaps simpler just not to mention these files, as they are optional? If a user was going to use them, they would know what they were. If they are following TSI workflows, they would read about these in the How-To guide.

[the remark this answers is not in the document]
Comment (neoformit) [reply webapp/home/labs/genome/annotation.yml]: Yes, if they're optional I'd just leave them out. The idea of listing input data here was to indicate what data the researcher NEEDS in order to run the tool - gets a bit complicated trying to list all possibilities. Shall I remove?

[the remark this answers is not in the document]
Comment (AnnaSyme) [reply webapp/home/labs/genome/annotation.yml]: yes plz remove thanks

**Pull request merged**